modular-docs-manual/content/topics/module_definition-procedure.adoc

```diff
@@ -0,0 +1,6 @@
+[[procedure-module-definition]]
+= Procedure Module Definition
+
+A procedure module is a "do" module. It gives the user numbered, step-by-step instructions.
```

Comment:
Similarly, I'd like to describe the concept module as an "understand" module. A reference module probably doesn't need a definition like this -- "reference" is a good word as it is.